### DISCUSSION TITLE
Patch to remove the in-video watermark

### DISCUSSION THREAD

**Discussion by Hyzian** [ReVanced/revanced-discussions#249] [Question] [answered]
So Vanced had this option to remove the little square at the bottom right corner of the video, and figured that it might not be too hard to patch so if it's possible then that would be great. 👍

I kinda miss this option as it left nothing but the video on the screen when watching in fullscreen. Thanks for all that the team has done, I believe in ReVanced. 🙏

![Screenshot_20220621-225846](https://user-images.githubusercontent.com/45723513/174904092-dff94237-61f0-4d5d-a320-83bfac520579.png)

**Comment by ronin08** [comment]
video link?

**Reply by Hyzian** [reply]
https://youtu.be/Xgz-AMx4WHU

**Reply by ronin08** [reply]
it's already patched! use official-build revanced before suggestions 
![Screenshot_2022-06-22-19-18-17-704_com vanced android youtube](https://user-images.githubusercontent.com/75478274/175046398-b576caa5-8ac2-4927-b52f-77d84d008b39.jpg)

**Reply by Hyzian** [reply]
So sorry. Which patch is it? And I am using the official, I built it myself in the cli.

**Reply by Hyzian** [reply]
is the patch official-build?
-i official-build?

**Reply by Hyzian** [reply]
> it's already patched! use official-build revanced before suggestions ![Screenshot_2022-06-22-19-18-17-704_com vanced android youtube](https://user-images.githubusercontent.com/75478274/175046398-b576caa5-8ac2-4927-b52f-77d84d008b39.jpg)

just asked in the discord and they checked the patch list. it aint there. are you using an unofficial build maybe?

my suggestion stays since i think it would be a great feature. :DD

**Reply by ronin08** [reply]
Use this: https://github.com/n0k0m3/revanced-build-template 
to build the revanced apk easily with latest available patches.

**Reply by jr1221** [reply]
@ronin08 What is the name of the patch?  I still see the watermark.

**Reply by ronin08** [reply]
Just use this to build:https://github.com/orgs/revanced/discussions/249#discussioncomment-3006674

**Reply by jr1221** [reply]
It appears you are fetching the revanced github patches repo, which has no such watermark support.  Unless you add patches, template patches = revanced patches.

**Reply by TheJeterLP** [reply]
Added with hide-watermark patch

**Comment by TheJeterLP** [comment, marked as answer]
Added with hide-watermark patch
